Announcement for 2022 (#241)
ournetworks/ournetworks.ca: index.md

redirect_from:
  - "/cfp/"
  - "/conference/"
---

_Drawing inspiration from our favourite sci-fi authors we’re headed to a future for Our Networks ~~2022~~ 2047. Join us over two weeks to call into the present an artefact from an Our Networks more than 20 years ahead. We will collectively narrate the history of the steps that allowed us to get there from here in order to understand where and how we can focus ourselves in the present._

> We've all been living through these weeks where decades happen. Looking back on our past we find neither dystopia nor utopia. Instead, our history winds a inbetween the two, with setbacks and sudden leaps toward the internet and web we want. In order to get there we’ve had to fight to transform the entire world. Today, that work is not over, and while there have been great strides in remaking where we live, work, and play as we tackle the climate crisis, there are still looming challenges and lingering concerns. 

Author: ```suggestion
We've all been living through these weeks where decades happen. Looking back on our past we find neither dystopia nor utopia. Instead, our history winds in-between, with setbacks and sudden leaps toward the internet and web we want. In order to even try and get there we’ve had to fight to transform the entire world. While there have been great strides in remaking where we live, work, and play as we tackle the interleaved crises of climate, capitalism, and co-option. There are still looming challenges and lingering concerns. 
```

> For an event that has been held alongside these struggles. Our Networks has been a place to share both victories and defeats together. Out of a pervading sense of large-scale ruin, we must keep inventing hope. But the kind of hope that isn’t optimism, rather the hope that is a discipline, one we practice every single day. In 2047 it feels like we are at another critical juncture, just like we were in 2022 and 2035. 

Author: ```suggestion
For an event that has been held alongside these struggles, Our Networks has been a place to share victories and defeats together. Out of a sense of large-scale ruin, we have kept inventing hope, the kind that isn’t optimism, but rather a discipline we practice every single day. In 2047 it feels like we are at another critical juncture, just like we were in 2022 and 2035. 
```
